src/components/Header/components/ProviderInfo/ProviderAccessible.tsx

```diff
@@ -1,18 +1,25 @@
 import { withStyles } from '@material-ui/core/styles'
-import Dot from '@material-ui/icons/FiberManualRecord'
 import * as React from 'react'
 
+import NetworkLabel from '../NetworkLabel'
 import CircleDot from 'src/components/Header/components/CircleDot'
 import Identicon from 'src/components/Identicon'
 import Col from 'src/components/layout/Col'
 import Paragraph from 'src/components/layout/Paragraph'
 import { shortVersionOf } from 'src/logic/wallets/ethAddresses'
+import WalletIcon from '../WalletIcon'
 import { connected as connectedBg, screenSm, sm } from 'src/theme/variables'
 
 const styles = () => ({
   network: {
     fontFamily: 'Averta, sans-serif',
   },
+  networkLabel: {
+    '& div': {
```

Comment:
there's padding already defined inside NetworkLabel, it also has some media queries in it. Maybe we should move it from there

Reply:
That's the reason why I did this. I want to override those styles just for this component. If I did it in NetworkLabel, I would change the styles everywhere this is used.